rulesets/NI.MinimumRecommendedRules.ruleset

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="NI.ruleset" Description="Code analysis rules for NI projects." ToolsVersion="14.0">
```

Comment:
Update Name

Reply:
I'm actually going to remove this file until a subsequent submission